source/server-selection/server-selection.rst

```diff
@@ -1431,6 +1475,39 @@ servers to come and go.  Making newly available servers either first or last
 could lead to unbalanced work.  Random selection has a better fairness
 guarantee and keeps the design simpler.
 
+operationCount-based selection within the latency window (multi-threaded or async)
```

Comment:
@p-mongo This section should contain the missing rationale you requested. Let me know if it is still missing anything. Also, I know you mentioned that the dimensionality of our algorithm was perhaps too high--this was similar feedback to what others on the review crew had, so we reduced the various different inputs down to just a single `operationCount` which should simplify this greatly.

Reply:
It might be nice to mention explicitly that we use operationCount as an approximation of load and that this algorithm is meant to distribute load more fairly than random.

Reply:
done